libsweep/include/protocol.h

```diff
@@ -136,11 +137,20 @@ static_assert(sizeof(response_info_version_s) == 21, "response info version size
 typedef struct {
   uint8_t cmdByte1;
   uint8_t cmdByte2;
+  uint8_t motor_ready[2];
+  uint8_t term;
+} response_info_motor_ready_s;
+
+static_assert(sizeof(response_info_motor_ready_s) == 5, "response info motor ready size mismatch");
```

Comment:
This means the SDK will no longer work on old devices, no? If so we need to increment the lib's major version.